kotlinx.fuzz.jazzer/src/main/kotlin/kotlinx/fuzz/jazzer/JazzerEngine.kt

```diff
@@ -36,14 +38,50 @@ class JazzerEngine(private val config: KFuzzConfig) : KFuzzEngine {
         )
     }
 
-    @OptIn(ExperimentalPathApi::class)
-    override fun runTarget(instance: Any, method: Method): Throwable? {
+    private fun configure(reproducerPath: Path): Pair<List<String>, Path> {
         val libFuzzerArgs = mutableListOf("fake_argv0")
         val corpusDir = createTempDirectory("jazzer-corpus")
 
         libFuzzerArgs += corpusDir.toString()
-        libFuzzerArgs += "-max_total_time=${config.maxSingleTargetFuzzTime.inWholeSeconds}"
         libFuzzerArgs += "-rss_limit_mb=${jazzerConfig.libFuzzerRssLimit}"
+        libFuzzerArgs += "-artifact_prefix=${reproducerPath.toAbsolutePath()}/"
+
+        Opt.keepGoing.setIfDefault(
```

Comment:
This line causes the build to crash, and it is reproduced on my computer if you run `./gradlew :kotlinx.fuzz.test:fuzz`.

The problem is that `OptItem::setIfDefault` method check whether we are trying to set the value of a property that has already been accessed before. When we are running multiple targets in one process --- we try to set the value of `keepGoing`  several times, which leads to Jazzer failing.

However, it feels like this problem should go away after you merge with main, as we are now running Jazzer in a separate process for each target

Reply:
fixed